### DISCUSSION TITLE
Upgrade to 3.7

### DISCUSSION THREAD

**Discussion by lasag-2511** [netbox-community/netbox#14805] [Help Wanted!]
Hi all,

I have upgraded netbox to latest version to 3.7.0 from 3.6 and after that I have the below issue with the scripts.

/opt/netbox/netbox/scripts/scan.py could not be loaded. 

Can anyone suggest me what could be ?

**Comment by candlerb** [comment]
That's exactly the same problem as you asked at #13627, and the solution was given to you there.

EDIT: the reason the upgrade triggered the problem is that the Netbox APIs can change from version to version, and you may need to update your custom scripts, reports, REST API integrations etc. to match. The most important changes are called out in the release notes, so check the [release notes for v3.7.0](https://github.com/netbox-community/netbox/releases/tag/v3.7.0).
